Bio/PDB/MMCIFParser.py

```diff
@@ -75,7 +75,6 @@ def _build_structure(self, structure_id):
             element_list = mmcif_dict["_atom_site.type_symbol"]
         except KeyError:
             element_list = None
-        seq_id_list = mmcif_dict["_atom_site.label_seq_id"]
```

Comment:
Removing this (and again later) puzzles me, but I am not familiar enough with the mmCIF format to understand the intent.

Reply:
In line 109-112 `seq_id_list` is set depending on which of `_atom_site.auth_seq_id` and `_atom_site.label_seq_id` are present. This line is therefore not used and causes an error if `_atom_site.label_seq_id` is not defined in the mmCIF file.

Technically this is separate to this PR but I put it here as I need this for the tests to pass (the mmCIF writer writes out `_atom_site.auth_seq_id` but not `_atom_site.label_seq_id` and hence reading the written files back in was erroring).